src/__test__/utils/cacheRequest.test.js

```diff
@@ -13,32 +13,54 @@ const fakeCacheKeyMappings = {
 
 const fakeData = {
   A: {
-    min: 0,
-    max: 6.8,
-    mean: 1.68,
-    stdev: 2.597331964,
-    expression: [0, 0, 0, 1.56, 0, 6.8, 3.4],
+    rawExpression: {
+      min: 0,
+      max: 6.8,
+      mean: 1.68,
+      stdev: 2.597331964,
```

Comment:
why do we have these here, but we don't have them in the violin test, where the violin plot is effectively looking at the same place in the store?

Reply:
Honestly I didn't look too much into the way the tests are written, just adapted them to the new structure so I don't know about the reasoning behind their designs.

Reply:
fair enough